applications/helpers.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+source `dirname "${BASH_SOURCE[0]}"`/../.helpers.sh
+
+create_application() {
+  create_type "application" "$@"
+}
+
+invoke_application() {
+  local type_name=$1
+  local input_data=$2
+  local expected_data=$3
+  local runtime=$4
+  invoke_type "application" $type_name "-d \"$input_data \""  $expected_data $runtime
```

Comment:
why is there a trailing space in the post body?

Reply:
shouldn't this also define the content type for the request?

Reply:
curl defaults to: `Content-Type: application/x-www-form-urlencoded`